src/geode/mesh/builder/surface_mesh_builder.cpp

```diff
@@ -155,9 +155,11 @@ namespace
         for( const auto e :
             geode::LRange{ surface.nb_polygon_edges( polygon_adj_id ) } )
         {
-            const geode::PolygonEdge adj_edge{ polygon_adj_id, e };
-            const auto adj_v0 = surface.polygon_vertex( adj_edge );
-            const auto adj_v1 = surface.polygon_edge_vertex( adj_edge, 1 );
+            absl::optional< geode::PolygonEdge > adj_edge{ absl::in_place,

```

Comment:
Is the in_place necessary ? I was under the impression that it was done directly with the {}

Reply:
the {} only works if the parameter is the same type as the type inside the optional. If you want to use other parameters to be forwarded to the constructor of the inside type, you need in_place